Source/BluetoothManager.swift

```diff
@@ -229,7 +229,7 @@ public class BluetoothManager {
      - parameter options: Dictionary to customise the behaviour of connection.
      - returns: Observable which emits next and complete events after connection is established.
      */
-    public func connectToPeripheral(peripheral: Peripheral, options: [String: AnyObject]? = nil)
+    public func connect(_ peripheral: Peripheral, options: [String: AnyObject]? = nil)
```

Comment:
I tried to follow the CoreBluetooth type signature here.
CBCentralManager:
`func connect(_ peripheral: CBPeripheral, 
     options: [String : Any]? = nil)`